streampipes-client-python/Makefile

```diff
@@ -0,0 +1,13 @@
+COMMIT_HASH := $(shell eval git rev-parse HEAD)
+
+format:
+	pre-commit run --all-files
+
+doc:
+	mkdocs build
+
+livedoc: doc
+	mkdocs serve --dirtyreload
+
+rebase:
+	git fetch && git rebase origin/master
```

Comment:
@kulbachcedric when is this rebase called? Sorry, I'm totally unaware of the python stuff ;-)

Reply:
> @kulbachcedric when is this rebase called? Sorry, I'm totally unaware of the python stuff ;-)

It's just for simplicity... We can remove it :-)

Reply:
We can keep it if it makes things easier...just wondering when this command would be running? 
@bossenti should this point to dev then?

Reply:
Ah yes, I our case it should point to `dev`.
It's just a convenient command that you can run locally. 
Nothing specific for python.